packages/kit/src/core/dev/plugin.js

```diff
@@ -1,373 +1,431 @@
+import { svelte } from '@sveltejs/vite-plugin-svelte';
 import fs from 'fs';
-import path from 'path';
-import { URL } from 'url';
 import colors from 'kleur';
+import path from 'path';
 import sirv from 'sirv';
+import { URL } from 'url';
+import { searchForWorkspaceRoot } from 'vite';
 import { installPolyfills } from '../../node/polyfills.js';
 import * as sync from '../sync/sync.js';
 import { getRequest, setResponse } from '../../node/index.js';
 import { SVELTE_KIT_ASSETS } from '../constants.js';
-import { get_mime_lookup, get_runtime_path, resolve_entry } from '../utils.js';
+import { get_aliases, get_mime_lookup, get_runtime_path, resolve_entry } from '../utils.js';
 import { coalesce_to_error } from '../../utils/error.js';
-import { load_template } from '../config/index.js';
+import { load_config, load_template, print_config_conflicts } from '../config/index.js';
 import { posixify } from '../../utils/filesystem.js';
 import { parse_route_id } from '../../utils/routing.js';
+import { deep_merge } from '../../utils/object.js';
 
 // Vite doesn't expose this so we just copy the list for now
 // https://github.com/vitejs/vite/blob/3edd1af56e980aef56641a5a51cf2932bb580d41/packages/vite/src/node/plugins/css.ts#L96
 const style_pattern = /\.(css|less|sass|scss|styl|stylus|pcss|postcss)$/;
 
 const cwd = process.cwd();
 
+/** @type {import('types').ValidatedConfig} */
+export const svelte_config = await load_config();
```

Comment:
this won't be reloaded when it changes. I don't think you can export `plugins`

Reply:
ah, so it's broken on `master` too then I guess?

Reply:
fixed